gulpfile.js

```diff
@@ -48,85 +52,98 @@ function formatByteMessage (source, data) {
-      .pipe(excludeModern.restore)
-
-      // * Process modern builds *
-      .pipe(excludeLegacy)
-      // Calculate size before autoprefixing
-      .pipe(bytediff.start())
-      // autoprefix modern builds
-      .pipe(postcss([autoprefixer({
-        env: 'modern'
-      })]))
-      // Write the amount gained by autoprefixing
-      .pipe(bytediff.stop(data => formatByteMessage('autoprefixer', data)))
-      .pipe(excludeLegacy.restore)
-
-      // Write the sourcemaps after making pre-minified changes
+      .pipe(postcss([postcssImport(cssImportOptions), postcssColorModFunction(), postcssInlineSvg()]))
+
+      .pipe(startDiff())
+      .pipe(postcss([postcssCssVariables({ preserve: true })]))
+      .pipe(endDiff('css variables'))
+
+      .pipe(startDiff())
+      .pipe(postcss([autoprefixer({ env: 'legacy' })]))
+      .pipe(endDiff('autoprefixer'))
+
       .pipe(sourcemaps.write('.'))
-      // Flatten output so files end up in dist/*, not dist/builds/*
-      .pipe(flatten())
-      // Write pre-minified styles
+      .pipe(flatten()) // Put files in dist/*, not dist/builds/*
       .pipe(gulp.dest(paths.styles.dest))
-      // Remove sourcemaps from the pipeline, only keep css
-      .pipe(filter('**/*.css'))
-      // Calculate size before minifying
-      .pipe(bytediff.start())
-      // Minify using cssnano, use extra-low precision while minifying inline SVGs
+
+      .pipe(filter('**/*.css')) // Remove sourcemaps from the pipeline
+
+      // <minifying>
+      .pipe(startDiff())
       .pipe(postcss([cssnano({ preset: ['default', { svgo: { floatPrecision: 0 } }] })]))
-      // Write the amount saved by minifying
-      .pipe(bytediff.stop(data => formatByteMessage('cssnano', data)))
-      // Rename the files have the .min suffix
+      .pipe(endDiff('minification'))
       .pipe(rename({ suffix: '.min' }))
-      // Write the sourcemaps after making all changes
+      // </minifying>
+
       .pipe(sourcemaps.write('.'))
-      // Write the minified files
       .pipe(gulp.dest(paths.styles.dest))
+      .pipe(gulp.dest(paths.docs.dest + '/water.css'))
+
+      .pipe(filter('**/*.css')) // Remove sourcemaps from the pipeline
       .pipe(sizereport({ gzip: true, total: false, title: 'SIZE REPORT' }))
-      // Stream any changes to browserSync
       .pipe(browserSync.stream())
   )
 }
 
-function watch () {
-  style()
+const docs = () => {
+  const htmlOnly = filter('**/*.html', { restore: true })
+  const jsOnly = filter('**/*.js', { restore: true })
+  const cssOnly = filter('**/*.css', { restore: true })
+
+  return (
+    gulp
+      // Exclude all HTML files but index.html
+      .src(paths.docs.src, { ignore: '**/!(index).html' })
 
-  browserSync.init({
-    server: {
-      baseDir: './'
-    },
-    startPath: 'index.html'
-  })
+      // * Process HTML *
+      .pipe(htmlOnly)
+      .pipe(posthtml([posthtmlInclude({ root: './docs/' }), htmlnano()]))
```

Comment:
We don't need `posthtml-include` anymore since we're not using any HTML modules via `<include href="">` anymore.
So we could remove this I think, but also it doesn't hurt there and gives us the option to modularize our HTML later if we want to. Opinions?

Reply:
I'm gonna remove it - it's very easy to add back later.